src/panels/config/storage/dialog-mount-view.ts

```diff
@@ -225,17 +254,41 @@ class ViewMountDialog extends LitElement {
     if (this._data?.name && !/^\w+$/.test(this._data.name)) {
       this._validationError.name = "invalid_name";
     }
+    if (
+      this._defaultBackupMount === null &&
+      this._data!.usage === SupervisorMountUsage.BACKUP &&
+      this._data!.default_backup_mount === undefined
+    ) {
+      this._data!.default_backup_mount = true;
+    }
   }
 
   private async _connectMount() {
     this._error = undefined;
     this._waiting = true;
+    const updateDefaultBackupMount = this._data!.default_backup_mount;
+    delete this._data!.default_backup_mount;
```

Comment:
if the updating or creating errors, we will now lose this

Reply:
Meh, yeah...
Will adjust it 👍